css/scrollbar-color/demos/multiple/index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Пример с несколькими разными скроллбарами — scrollbar-color — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap">
+  <style>
+    *, *::before, *::after {
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+    }
+
+    html {
+      scrollbar-color: #2E9AFF #F498AD;
+    }
+
+    body {
+      min-height: 100vh;
+      padding: 50px;
+      background-color: #18191C;
+      color: #FFFFFF;
+      font-family: "Roboto", sans-serif;
+    }
+
+    @media (max-width: 768px) {
+      body {
+        padding: 30px;
+      }
+    }
+
+    p {
+      font-size: 30px;
+      margin-bottom: 2em;
+    }
+
+    .container-400 {
+      height: 400px;
+      overflow: scroll;
+      scrollbar-color: rgb(223, 139, 139) rgb(230, 243, 118);
```

Comment:
```suggestion
      scrollbar-color: rgb(223 139 139) rgb(230 243 118);
```

Мы стараемся использовать современный синтаксис записи цветов. Кстати, тут нет альфа-канала. Нам точно нужен здесь RGB или можно переписать на HEX?

Reply:
можно, это я не обратил внимание что в вскод они проставились как ргб, поменял